documentation/htc_workloads/managing_data/data-transfer.md

```diff
@@ -0,0 +1,199 @@
+
+| File Location  | Job Input/ Output Data Sizes   | Transfer to/from Jobs | Availability |
+| :---------- | :----------------------------------- | :------ | :------ | Files available only to your jobs. |
+| `/home`    | Less than 500 MB per job  | **Input Files**:`transfer_input_files = InFile.txt`<br>**Output Files**: Files created in job return to `/home` by default. |
+| `/protected` | Greater than 500 MB per job | **Input Files**: `transfer_input_files = stash:///ospool/protected/user/InFile.txt`<br>**Output Files**: `transfer_output_remaps = stash:///stash:///ospool/protected/user/OutFile.txt` |
+
+
+
+
+**Important Details:** 
+* Data stored within `/home` and `/protected` is available only to your jobs. 
+* Large files stored in `/protected` are cached, so it is important to use a descriptive file name (possibly using version names or dates within the file name) to ensure you know what version of the file you are using within your job. 
+
+
+## Transfer Small Job Input and Output Files to/from /home 
+/home: 
+  * Only job input and output data less than 500 MB should be stored in `/home`
+  * *Job Input files*: use HTCondor's `transfer_input_files` submit file option with either absolute or relative paths to your input files. For example, `transfer_input_files = /home/<username>/InFile.txt`
+  * *Job Output files*: By default, files created by your job will automatically be returned to your `/home` directory. If you would like a file to return to a diffrent subfolder within your `/home` directory, use HTCondor's `transfer_output_remaps` option. For more information, see {Guide}
+
+## Transfer Large Job Input and Output Files to/from /protected 
+/protected: 
+  * Only job input and output data greater than 500 MB should be stored in `/protected`
+  * *Job Input files*: Transfer files from /protected using `transfer_input_files = stash:///ospool/protected/<username>/InFile.txt`. This combines HTCondor's `transfer_input_files` with a file transfer protocol, `stash:///ospool/protected/<username>/`. 
+  * *Job Output files*: If you would like a job to transfer a large file back to your /protected directory, in your HTCondor submit file, use `transfer_output_remaps = stash:///ospool/protected/<username>/OutFile.txt`. 
+  
+
+# HTCondor's Transfer_output_remaps 
+
+For output, users can use the `transfer_output_remaps` option within their job's submit file, which will transfer the user's specified file to a different location than the default `/home/username`. 
+
+By using `transfer_output_remaps`, it is possible to specify what path to save a file to and what name to save it under. Using this approach, it is possible to save files back to specific locations in `/public` (as well as your `/home` directory, if desired).
+
+The syntax for transfer_output_remaps is:
+
+```
+transfer_output_remaps = "Output.txt = path/to/save/file/under/output.txt; Output.txt = path/to/save/file/under/RenamedOutput.txt"
+```
```

Comment:
```suggestion
```
To save more than one file using `transfer_output_remaps`, seporate your list of output files by a semicolon like so: 

```
transfer_output_remaps = "Output.txt = path/to/save/file/under/Output.txt; Results.txt = path/to/save/file/under/Results.txt"
```

To have HTCondor rename a job's output file (Output.txt), use `transfer_output_remaps` like so:

```
transfer_output_remaps = "Output.txt = path/to/save/file/under/RenamedOutputFile.txt"
```